lnbits/extensions/usermanager/crud.py

```diff
@@ -92,28 +87,28 @@ async def create_usermanager_wallet(
     return wallet_created
 
 
-async def get_usermanager_wallet(wallet_id: str) -> Optional[Wallets]:
+async def get_usermanager_wallet(wallet_id: str) -> Optional[Wallet]:
     row = await db.fetchone(
         "SELECT * FROM usermanager.wallets WHERE id = ?", (wallet_id,)
     )
-    return Wallets(**row) if row else None
+    return Wallet(**row) if row else None
 
 
-async def get_usermanager_wallets(admin_id: str) -> Optional[Wallets]:
+async def get_usermanager_wallets(admin_id: str) -> list[Wallet]:
```

Comment:
- should be `List[Wallet]` - uppercase `List` instead of `list`

Reply:
36ae851a